backend/pom.xml

```diff
@@ -211,6 +211,7 @@
         <!-- <module>extensionsupport</module> -->
 
         <module>de-metas-salesorder</module>
+        <module>de.metas.postfinance</module>
```

Comment:
please insert it further up, where it belongs alphabetically

Reply:
done